lib/v2/mdpi/journal.js

```diff
@@ -28,7 +28,7 @@ module.exports = async (ctx) => {
             const title = $2(item).find('.title-link').text();
             const link = `${host}${$2(item).find('.title-link').attr('href')}`;
             const authors = $2(item).find('.authors').find('.inlineblock').text();
-            const doi = $2(item).find('.color-grey-dark').find('a').attr('href').replace('https://doi.org/', '');
+            const doi = $2(item).find('.color-grey-dark').find('a').attr('href');
```

Comment:
If you visit `/mdpi/analytica?scihub=1`
The link will now become `https://sci-hub.se/https://doi.org/xx.xxxxx/xxxxxxxx` if you don't replace doi

Reply:
Ok, I will resolve it.